src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexBoyerMoore.cs

```diff
@@ -347,26 +347,29 @@ public string Dump(string indent)
         {
             StringBuilder sb = new StringBuilder();
 
-            sb.Append(indent + "BM Pattern: " + Pattern + "\n");
+            sb.Append(indent + "BM Pattern: " + Pattern);
+            sb.AppendLine();
```

Comment:
```suggestion
            sb.AppendLine(indent + "BM Pattern: " + Pattern);
```
We don't care about perf here, but it could also be:
```C#
sb.Append(indent).Append("BM Pattern: ").AppendLine(Pattern);
```
or since we don't care about perf:
```C#
sb.AppendLine($"{indent}BM Pattern: {Pattern}");
```